src/main/java/org/usfirst/frc4904/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,83 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem;
+import org.usfirst.frc4904.standard.subsystems.SolenoidSubsystem.SolenoidState;
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+
+public class Intake extends SubsystemBase {
+  private static final double defaultOnSpeed = 0.7;
+  private static final double defaultOffSpeed = 0.0;
```

Comment:
For now, I would probably set these to .5 and 0. While it may be a personal preference, I think it's worth making it clear that .7 is the untested, theorized speed.